src/argilla/server/models.py

```diff
@@ -34,26 +34,54 @@ def default_inserted_at(context):
     return context.get_current_parameters()["inserted_at"]
 
 
+class AnnotationType(str, Enum):
+    text = "text"
+    single_label = "single_label"
```

Comment:
So, after the slack thread, we can change this to:
```suggestion
    free_text = "free_text"
    rating = "rating"
```

Reply:
I have added `rating` but didn't change `text` to `free_text`. Not sure if `free_text` it's better than simply `text`.

Reply:
I agree that simply `text` is better, the free doesn't add anything

Reply:
Sorry, I reviewed this quickly, and I was thinking about single and multi-label when I did the changes. `text` fits well for me too.